how-to/customize-workspace/client/src/home.ts

```diff
@@ -75,6 +75,14 @@ function mapAppEntriesToSearchEntries(apps: App[]): HomeSearchResult[] {
 				entry.label = "View";
 				entry.actions = [action];
 			}
+			if (apps[i].manifestType === "window" || apps[i].manifestType === "inline-window") {
```

Comment:
Shouldn't these be `else if` or a `switch`

Reply:
Updated to use a switch